spec.emu

```diff
@@ -21,110 +21,45 @@ contributors: Jordan Harband
 	</emu-alg>
 </emu-clause>
 
-<emu-clause id="sec-promisereaction-records">
-	<h1>PromiseReaction Records</h1>
-	<p>…</p>
-	<emu-table id="table-58" caption="PromiseReaction Record Fields">
-		<table>
-			<tbody>
-			<tr>
-				<th>
-					Field Name
-				</th>
-				<th>
-					Value
-				</th>
-				<th>
-					Meaning
-				</th>
-			</tr>
-			<tr>
-				<td>
-					…
-				</td>
-				<td>
-					…
-				</td>
-				<td>
-					…
-				</td>
-			</tr>
-			<tr>
-				<td>
-					[[Type]]
-				</td>
-				<td>
-					Either `"Fulfill"`, `"Reject"`, or `"Finally"`.
-				</td>
-				<td>
-				</td>
-			</tr>
-			<tr>
-				<td>
-					…
-				</td>
-				<td>
-					…
-				</td>
-				<td>
-					…
-				</td>
-			</tr>
-			</tbody>
-		</table>
-	</emu-table>
-</emu-clause>
-
 <emu-clause id="sec-performpromisefinally" aoid="PerformPromiseFinally">
 	<h1>PerformPromiseFinally ( _promise_, _onFinally_, _resultCapability_ )</h1>
```

Comment:
I wanted to keep parity with PerformPromiseThen, since they're both similar. I could certainly inline it though, if you think it'd be better.